controllers/openstacknetconfig_controller.go

```diff
@@ -388,16 +403,26 @@ func (r *OpenStackNetConfigReconciler) applyNetAttachmentConfig(
 
 	op, err := controllerutil.CreateOrUpdate(context.Background(), r.Client, attachConfig, apply)
 	if err != nil {
-		err = common.WrapErrorForObject(fmt.Sprintf("Updating %s OpenStackNetworkAttach", attachConfig.Name), attachConfig, err)
+		cond.Message = fmt.Sprintf("Failed to create or update %s %s ", attachConfig.Kind, attachConfig.Name)
+		cond.Reason = ospdirectorv1beta1.ConditionReason(ospdirectorv1beta1.OpenStackProvisionServerCondReasonCreateError)
```

Comment:
Looks like some left-overs here from copy-paste: `OpenStackProvisionServerCondReasonCreateError`

Reply:
fixes in latest commit